e2e/scripts/actions/env_path.test

```diff
@@ -0,0 +1,30 @@
+#!/singularity/test
+# env path tests
+
+source common.func
+
+default_image="docker://alpine:3.8"
```

Comment:
I think these defaults and custom options should be inputs to the script/test so that we can test many different containers here. A table-driven approach with a loop would be how we would do it in `go`. How would we accomplish that here?

Reply:
Well I just ported current tests as it were written in Go. Here we could use same approach with a bash array